LEAF_Request_Portal/admin/templates/mod_groups.tpl

```diff
@@ -169,7 +169,7 @@ function addMember(groupID, userID) {
 
 // convert to title case
 function toTitleCase(str) {
-    return str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
+    return (str == "" || str == null) ? "" : str.replace(/\w\S*/g, function(txt){return txt.charAt(0).toUpperCase() + txt.substr(1).toLowerCase();});
```

Comment:
Does this tie directly to the email template editor line number overlap? This template file shouldn't be related to the email template editor.